messenger.rst

```diff
@@ -1223,6 +1223,8 @@ Then, make sure to "route" your message to *both* transports:
 
 That's it! You can now consume each transport:
 
+It's also possible to specify `from_transport` as attribute on the `messenger.message_handler` tag.
```

Comment:
The previous sentence ends with a `:`, so putting your sentence between the previous one and the code example looks wrong to me.

I would propose to add:

```rst
.. note::

    It is possible to specify ``from_transport`` as attribute on the ``messenger.message_handler`` tag.

    .. versionadded:: 4.4
        
        The ability to specify ``from_transport`` on the ``messenger.message_handler`` tag
        itself, was added in Symfony 4.4.
```

@javiereguiluz / @xabbuh would you agree to add the `versionadded` to the note itself?

Reply:
Applied the change.